kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -1,4 +1,27 @@
 package com.kodilla.testing.collection;
 
+import java.util.LinkedList;
+import java.util.List;
+
 public class OddNumbersExterminator {
+
+
+
+    public OddNumbersExterminator(){ }
```

Comment:
Ten konstruktor jest do wyrzucenia. To jest domyslny konstruktor i jest niepotrzebne, aby jawnie go deklarowac